### PR TITLE
Make JAX-B API optional

### PR DESCRIPTION
**Pull request by jansupol**
Signed-off-by: Jan Supol <jan.supol@oracle.com>

### CODE REVIEW

File: core-server/pom.xml [hunk 1]
```diff
@@ -198,7 +192,18 @@
             <artifactId>org.osgi.core</artifactId>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
```
Comment (senivam) [inline core-server/pom.xml:195]: please consider changing copyright year in this file

File: connectors/jetty-connector/pom.xml [hunk 2]
```diff
@@ -41,6 +41,12 @@
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-client</artifactId>
         </dependency>
+        <dependency>
```
Comment (senivam) [inline connectors/jetty-connector/pom.xml:44]: please consider changing copyright year in this file

File: examples/bookstore-webapp/pom.xml [hunk 3]
```diff
@@ -63,6 +63,10 @@
             <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
             <version>${jstl.version}</version>
         </dependency>
+        <dependency>
```
Comment (senivam) [inline examples/bookstore-webapp/pom.xml:66]: please consider updating copyright year in this file

File: examples/https-clientserver-grizzly/pom.xml [hunk 4]
```diff
@@ -49,6 +49,10 @@
             <groupId>org.glassfish.jersey.inject</groupId>
             <artifactId>jersey-hk2</artifactId>
         </dependency>
+        <dependency>
```
Comment (senivam) [inline examples/https-clientserver-grizzly/pom.xml:52]: please consider updating copyright year in this file

File: examples/jaxb/pom.xml [hunk 5]
```diff
@@ -67,6 +67,10 @@
                 <jdk>[11,)</jdk>
             </activation>
             <dependencies>
+                <dependency>
```
Comment (senivam) [inline examples/jaxb/pom.xml:70]: please consider updating copyright year in this file

File: examples/multipart-webapp/pom.xml [hunk 6]
```diff
@@ -37,6 +37,10 @@
             <groupId>org.glassfish.jersey.inject</groupId>
             <artifactId>jersey-hk2</artifactId>
         </dependency>
+        <dependency>
```
Comment (senivam) [inline examples/multipart-webapp/pom.xml:40]: please consider updating copyright year in this file

File: ext/bean-validation/pom.xml [hunk 7]
```diff
@@ -97,6 +97,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```
Comment (senivam) [inline ext/bean-validation/pom.xml:100]: please consider updating copyright year in this file

File: ext/bean-validation/src/main/java/org/glassfish/jersey/server/validation/ValidationErrorData.java [hunk 8]
```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 20208 Oracle and/or its affiliates. All rights reserved.
```
Comment (senivam) [inline ext/bean-validation/src/main/java/org/glassfish/jersey/server/validation/ValidationErrorData.java:2]: please consider fixing copyright year in this file

File: ext/entity-filtering/pom.xml [hunk 9]
```diff
@@ -34,6 +34,11 @@
     </description>
 
     <dependencies>
+        <dependency>
```
Comment (senivam) [inline ext/entity-filtering/pom.xml:37]: please consider updating copyright year in this file

**Review by senivam — CHANGES_REQUESTED** [review]

**Review by senivam — APPROVED** [review]
LGTM

**Review by Verdent — APPROVED** [review]

**Pull request merged**